src/main/java/org/tarantool/TarantoolClientImpl.java

```diff
@@ -115,12 +115,6 @@ private void initClient(SocketChannelProvider socketProvider, TarantoolClientCon
         this.syncOps = new SyncOps();
         this.composableAsyncOps = new ComposableAsyncOps();
         this.fireAndForgetOps = new FireAndForgetOps();
-        if (config.useNewCall) {
```

Comment:
It worth to raise an error for `config.useNewCall == false` at least for year I think.

Reply:
it's slightly strange to give an option to use an old function and fail an initialization. Maybe, to log a warning is the more appropriate thing here?  Say, 'Call operation v1.6 is deprecated. New call operation will be used instead.' Or if we don't care about a breaking change here, we can completely eliminate this option from the config.

Reply:
If we'll remove the config option at all a user will receive an error too, so this is okay for me. Leave the option and call_16 support for now seems okay for me too, but I would change the default to call_17 (+warnings, yep). In other words, I'm okay with both variants.

Reply:
Ok. The call_16 left but is not a default value anymore. A new call is used by default now.